packages/flutter_tools/test/general.shard/windows/visual_studio_test.dart

```diff
@@ -146,73 +136,80 @@ void main() {
     VisualStudio visualStudio;
 
     setUp(() {
-      mockProcessManager = MockProcessManager();
+      processManager = FakeProcessManager.list(<FakeCommand>[]);
+      fileSystem = MemoryFileSystem.test(style: FileSystemStyle.windows);
+      logger = BufferLogger.test();
+      visualStudio = VisualStudio(
+        fileSystem: fileSystem,
+        platform: windowsPlatform,
+        logger: logger,
+        processManager: processManager,
+      );
     });
 
     testUsingContext('isInstalled returns false when vswhere is missing', () {
```

Comment:
Do we still use testUsingContext when not adding any overrides?

Reply:
Removed